index.Rmd

```diff
@@ -286,7 +290,7 @@ In order to achieve these objectives, you should try to follow a few good practi
     - Consistent placement of curly braces.
 * [Make your own packages](#__optional__Writing_packages_).
 * Keep a similar [directory structure](../RProjectManagement/index.html) across your projects.
-* Use [version control](../RProjectManagement/index.html) to develop/maintain your projects and packages.
```

Comment:
The link currently brings students to the start of tutorial 1. Could be more helpful to bring them to the version control section directly. Check that link works.